roles/osp_observability/defaults/main.yml

```diff
@@ -1,7 +1,16 @@
 ---
-
-libdir: /usr/share/osp-observability
-workdir: /var/lib/osp-observability
-
 openstack_cloud_name: overcloud
+prometheus_config_basedir: ""
+prometheus_remote_write:
+  stf:
+    url: https://default-prometheus-proxy-service-telemetry.apps.FAKE.ocp.cluster/api/v1/write
+    basic_user: internal
+    basic_pass: Pl4iNt3xTp4a55
+    ca_cert: |
+      -----BEGIN CERTIFICATE-----
+      ABCDEFGHIJKLMNOPQRSTUVWXYZ
+      -----END CERTIFICATE-----
+prometheus_scrape_endpoints:
+  collectd:
+    - "localhost:9103"
```

Comment:
Well I now see there will be some defaults enabled, so some of my suggestions in #18 about adding `is defined` and `length > 0` may not be necessary.